README.md

```diff
@@ -19,7 +19,7 @@ to profile your application, use specific minimal library:
 
 XHGui has the following requirements:
 
-- Known to work: PHP >= 7.2, 8.0, 8.1
+- Known to work: PHP >= 7.2, 8.0-8.3
```

Comment:
composer.json now requires php > 8.0
Should we remove the mention of PHP 7.2?

Reply:
I think we also removed the test for PHP 7.2/7.3

Reply:
https://github.com/perftools/xhgui/commit/88fc190bfa2f8a7f0fbb09170d95691fd583106c